.github/workflows/ADD_TO_HTO_PROJECT.yml

```diff
@@ -0,0 +1,20 @@
+name: ADD_TO_PROJECT
+
+on:
+  issues:
+    types:
+      - opened
+      - transferred
+  pull_request:
+    types:
+      - opened
```

Comment:
In the desktop modeler team, In the past, PRs often include more discussions and context than the original issue. We also usually assign the DRI to the PR itself, so it's more present on the taskboard. All to really track it, as PRs are important to report the current workload.

It also made sense to track them to not miss open PRs that could be stale otherwise (e.g. if no issue is attached, e.g. dependency updates, some community contributions). It kind of forced us to have a look regularly, especially because it was polluting our review column.